### ISSUE TITLE
Pre-Populate Value for entries dropdown throws error

### ISSUE THREAD

**Issue by jaydensmith** [verbb/formie#1931]
### Describe the bug

I have an entries dropdown which when populated from a query param (i.e. `?course=1200`), it throws a 500 error.

### Form settings

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

### Craft CMS version

5.2.0

### Plugin version

3.0.0-beta.14

### Multi-site?

No

### Additional context

Trace:
```
yii\base\ErrorException: Array to string conversion in /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Db.php:608
Stack trace:
#0 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/ErrorHandler.php(79): yii\base\ErrorHandler->handleError(2, 'Array to string...', '/Users/jaydensm...', 608)
#1 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Db.php(608): craft\web\ErrorHandler->handleError(2, 'Array to string...', '/Users/jaydensm...', 608)
#2 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Db.php(873): craft\helpers\Db::parseParam('elements.id', Array, '=', false, 'integer')
#3 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1577): craft\helpers\Db::parseNumericParam('elements.id', Array)
#4 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/elements/db/ElementQuery.php(2064): craft\elements\db\ElementQuery->prepare(Object(craft\db\mysql\QueryBuilder))
#5 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1891): craft\elements\db\ElementQuery->prepareSubquery()
#6 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/base/ElementField.php(140): craft\elements\db\ElementQuery->exists()
#7 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/base/ElementField.php(404): verbb\formie\base\ElementField->isValueEmpty(Object(craft\elements\db\EntryQuery), NULL)
#8 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Extension/CoreExtension.php(1635): verbb\formie\base\ElementField->getDisplayTypeValue(Object(craft\elements\db\EntryQuery))
#9 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Template.php(141): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\fields\Entries), 'getDisplayTypeV...', Array, 'method', false, false, false, 7)
#10 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/8a/8a7f449cbac3ec5af6bbc4d5b9a2a1f8.php(48): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\fields\Entries), 'getDisplayTypeV...', Array, 'method', false, false, false, 7)
#11 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_e604fe9c8f4d095a06326d0b1af52219->doDisplay(Array, Array)
#12 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#13 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#14 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render(Array)
#15 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Environment.php(280): Twig\TemplateWrapper->render(Array)
#16 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/View.php(488): Twig\Environment->render('formie/_special...', Array)
#17 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/elements/Form.php(1106): craft\web\View->renderTemplate('formie/_special...', Array, 'cp')
#18 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/base/Field.php(1108): verbb\formie\elements\Form->renderTemplate(Array, Array)
#19 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Extension/CoreExtension.php(1635): verbb\formie\base\Field->getFrontEndInputHtml(Object(verbb\formie\elements\Form), Object(craft\elements\db\EntryQuery), Array)
#20 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Template.php(141): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\fields\Entries), 'getFrontEndInpu...', Array, 'method', false, false, false, 22)
#21 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/06/060829d6f9f3f733907a7083cf7bf7ee.php(96): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\fields\Entries), 'getFrontEndInpu...', Array, 'method', false, false, false, 22)
#22 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_2121922904c46ebbed8604f37fe38077->doDisplay(Array, Array)
#23 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#24 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#25 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render(Array)
#26 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Environment.php(280): Twig\TemplateWrapper->render(Array)
#27 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/View.php(488): Twig\Environment->render('formie/_special...', Array)
#28 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/elements/Form.php(1106): craft\web\View->renderTemplate('formie/_special...', Array, 'cp')
#29 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/services/Rendering.php(192): verbb\formie\elements\Form->renderTemplate(Array, Array)
#30 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/variables/Formie.php(91): verbb\formie\services\Rendering->renderField(Object(verbb\formie\elements\Form), Object(verbb\formie\fields\Entries), Array)
#31 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Extension/CoreExtension.php(1635): verbb\formie\variables\Formie->renderField(Object(verbb\formie\elements\Form), Object(verbb\formie\fields\Entries), Array)
#32 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Template.php(141): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderField', Array, 'method', false, false, false, 17)
#33 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/0b/0b584a08a0c7b0babb562b373a3e0108.php(81): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderField', Array, 'method', false, false, false, 17)
#34 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_8ccb8c8104f8913c97c714ba14a7bfd2->doDisplay(Array, Array)
#35 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#36 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#37 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render(Array)
#38 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Environment.php(280): Twig\TemplateWrapper->render(Array)
#39 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/View.php(488): Twig\Environment->render('formie/_special...', Array)
#40 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/elements/Form.php(1106): craft\web\View->renderTemplate('formie/_special...', Array, 'cp')
#41 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/services/Rendering.php(152): verbb\formie\elements\Form->renderTemplate(Array, Array)
#42 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/variables/Formie.php(86): verbb\formie\services\Rendering->renderPage(Object(verbb\formie\elements\Form), Object(verbb\formie\models\FieldLayoutPage), Array)
#43 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Extension/CoreExtension.php(1635): verbb\formie\variables\Formie->renderPage(Object(verbb\formie\elements\Form), Object(verbb\formie\models\FieldLayoutPage), Array)
#44 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Template.php(141): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderPage', Array, 'method', false, false, false, 92)
#45 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/1a/1a48337d3590b8536d3ba98f151663ca.php(259): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderPage', Array, 'method', false, false, false, 92)
#46 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_a1dedf0f859b6cff9feb70caac9c687c->doDisplay(Array, Array)
#47 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#48 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#49 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render(Array)
#50 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Environment.php(280): Twig\TemplateWrapper->render(Array)
#51 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/View.php(488): Twig\Environment->render('formie/_special...', Array)
#52 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/elements/Form.php(1106): craft\web\View->renderTemplate('formie/_special...', Array, 'cp')
#53 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/services/Rendering.php(85): verbb\formie\elements\Form->renderTemplate(Array, Array)
#54 /Users/jaydensmith/public_html/x/vendor/verbb/formie/src/variables/Formie.php(81): verbb\formie\services\Rendering->renderForm(Object(verbb\formie\elements\Form), Array)
#55 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Extension/CoreExtension.php(1635): verbb\formie\variables\Formie->renderForm(Object(verbb\formie\elements\Form), Array)
#56 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/helpers/Template.php(141): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array, 'method', false, false, false, 238)
#57 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/4b/4b607d3fc3e6b028e4ae3e681cedb446.php(49): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array, 'method', false, false, false, 238)
#58 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_39bb17f2cb507b4746b4ed8ebbfaaa2d->doDisplay(Array, Array)
#59 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#60 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/7f/7faee12a9141dae1f3397ce9453b712c.php(38): Twig\Template->display(Array)
#61 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_e7aca09907d45cb618c21687ad64e39c->doDisplay(Array, Array)
#62 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#63 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/a0/a007b6d7bc47e4a86dd7c48d20b0880f.php(66): Twig\Template->display(Array)
#64 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_648eba5ef0a8c2203d46b0bc6cd6d847->doDisplay(Array, Array)
#65 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#66 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/22/223196e00cdcabf6249640f175c4f6ad.php(62): Twig\Template->display(Array)
#67 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_f28b6bfec80364c9a9c2d7deb97d5e2f->doDisplay(Array, Array)
#68 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#69 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/a0/a007b6d7bc47e4a86dd7c48d20b0880f.php(66): Twig\Template->display(Array)
#70 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_648eba5ef0a8c2203d46b0bc6cd6d847->doDisplay(Array, Array)
#71 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#72 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/fc/fcf05a64ffda1ef504f1df92837347b2.php(59): Twig\Template->display(Array)
#73 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(171): __TwigTemplate_43322e053aaf66cdd272f6976698dbdb->block_content(Array, Array)
#74 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/df/df5fb5cd67642c18ad0071bd43096be3.php(245): Twig\Template->displayBlock('content', Array, Array)
#75 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_32904b3e51e5355095494bdafff75f17->doDisplay(Array, Array)
#76 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#77 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/fc/fcf05a64ffda1ef504f1df92837347b2.php(43): Twig\Template->display(Array, Array)
#78 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_43322e053aaf66cdd272f6976698dbdb->doDisplay(Array, Array)
#79 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#80 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->display(Array, Array)
#81 /Users/jaydensmith/public_html/x/storage/runtime/compiled_templates/c7/c716a720a769ba5f86547315231f1598.php(43): Twig\TemplateWrapper->display(Array)
#82 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(394): __TwigTemplate_6ee1164c9a0b898e288670cd669465a6->doDisplay(Array, Array)
#83 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#84 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#85 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/TemplateWrapper.php(38): Twig\Template->render(Array)
#86 /Users/jaydensmith/public_html/x/vendor/twig/twig/src/Environment.php(280): Twig\TemplateWrapper->render(Array)
#87 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/View.php(488): Twig\Environment->render('pages/index', Array)
#88 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/View.php(541): craft\web\View->renderTemplate('pages/index', Array)
#89 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(57): craft\web\View->renderPageTemplate('pages/index', Array, 'site')
#90 /Users/jaydensmith/public_html/x/vendor/yiisoft/yii2/web/Response.php(1109): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#91 /Users/jaydensmith/public_html/x/vendor/craftcms/cms/src/web/Response.php(338): yii\web\Response->prepare()
#92 /Users/jaydensmith/public_html/x/vendor/yiisoft/yii2/web/Response.php(340): craft\web\Response->prepare()
#93 /Users/jaydensmith/public_html/x/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
#94 /Users/jaydensmith/public_html/x/public_html/index.php(25): yii\base\Application->run()
#95 /Users/jaydensmith/.composer/vendor/laravel/valet/server.php(110): require('/Users/jaydensm...')
#96 {main}
```

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.14"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.15](https://github.com/verbb/formie/releases/tag/3.0.0-beta.15)
